phase3/Kconfig

```diff
@@ -40,4 +40,10 @@ config phase3.weave_net
 	  Should we deploy weave-net? You likely only want this if using kubeadm for phase2.
 endif
 
+config phase3.cni
+	string "CNI plugin to use"
+	default weave
```

Comment:
As you might suspect, that's a pretty big question. There's a bit of documentation [here](https://github.com/kubernetes/test-infra#create-a-new-job), and I pasted some more links on the sig-cluster-lifecycle Slack channel. If those all don't suffice, it might be easiest to do a screensharing video chat (maybe an ad-hoc Zoom meeting?) to walk through the layers and answer questions.